src/main/resources/templates/index.html

```diff
@@ -7,50 +7,42 @@
     <meta name="author" content="Wesley Reed">
     <title>Inventory Manager</title>
 
+    <!-- Bootstrap core CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <!--Boostrap core CSS end -->
+
+    <nav th:replace="fragments/inlineStyles :: styles"></nav>
+</head>
+<body>
+
+    <nav th:replace="fragments/navigation :: navbar"></nav>
 
-    <!--Dan's stuff-->
-    <p th:text="${item}" />
     <form action="#" th:action="@{/saveItem}" th:object="${item}" method="GET">
         <label>Item name:</label>
-        <input type="text" name="name" th:field="*{name}"/><br><br>
+        <input type="text" name="name" th:placeholder="*{getItemName}"/><br><br>
         <label>Item Description:</label>
-        <input type="text" name="description" th:field="*{description}"/><br><br>
-        <label>Location:</label>
-        <input type="text" name="location" th:field="*{location}"/><br><br>
+        <input type="text" name="description" th:placeholder="*{getItemDescription}"/><br><br>
         <label>Location:</label>
-        <input type="text" name="assigned_to" th:field="*{assignedTo}"/><br><br>
-        <label>Location:</label>
-        <input type="text" name="status_id" th:field="*{statusId}"/><br><br>
-        <label>Item ID</label>
-        <input type="text" name="id" th:field="*{id}"/><br><br>
+        <input type="text" name="location" th:placeholder="*{getItemLocation}"/><br><br>
+        <label>Assigned To:</label>
+        <input type="text" name="assigned_to" th:placeholder="*{assignedTo}"/><br><br>
+        <label>Status:</label>
+        <input type="text" name="status_id" th:placeholder="*{statusId}"/><br><br>
+        <label>Item ID:</label>
+        <input type="text" name="id" th:placeholder="*{getItemId}"/><br><br>
         <button name="submit" type="submit" value="0">Submit</button>
     </form>
```

Comment:
Moved Dan's forms out of the head and under the navigation bar. This is UI for your logic and should be what is in the body of the page. Also after doing a bit of research, the only tags that _should_ go into the head are:

- meta
- link
- title
- style
- script
- noscript
- base

Source: https://htmlhead.dev/#:~:text=Valid%20elements%20include%20meta,search%20engines%2C%20bots%2C%20etc.

Reply:
Correct.. these are the only tags that should be in head. Move Dan's form outside of it, and into body.